src/web/modules/stripe/basic/basic.http.ts

```diff
@@ -98,7 +101,24 @@ const submitAccountCreate = async function submitAccountCreate(
   }
 }
 
+const logStripeBalance = async function logStripeBalance(
+  req: Request,
+  res: Response
+): Promise<void> {
+  if (!STRIPE_ACCOUNT_API_KEY) {
+    throw new CustomValidationError('Stripe API Key was not configured for this Toolbox instance')
+  }
+  const balance = await stripeClient.getStripeApi().balance.retrieve()
+  const available = balance.available.shift()
+  logger.info('Stripe balance retrieved.', {
+    'amount': JSON.stringify(available.amount),
+    'currency': JSON.stringify(available.currency)
+  })
```

Comment:
Unless I've missed something I think we're OK to not stringify here, amount looks like a number and currency a string both of which should be fine to log.
```suggestion
  logger.info('Stripe balance retrieved.', {
    amount: available.amount,
    currency: available.currency
  })
```

Or if you want to go full JS
```suggestion
  const { amount, currency } = balance.available.shift()
  logger.info('Stripe balance retrieved.', { amount, currency })
```

Reply:
Oh amazing. I used `JSON.stringify` because when I originally did `balance.available` and variants on that, for e.g. `balance.available.amount` and logged it I was literally logging `[JSONObject]` or something like that.